code/engine.h

```diff
@@ -5,6 +5,7 @@
 
 typedef struct EngineStruct {
   uint16_t rpm;                ///< [rpm] Engine rotation
+  uint16_t engineReport;    ///< Time to report engine Status [S]

```

Comment:
Suggestion: rename and replace with `time_t` from `time.h`. This is usually the standard type used to store time in C.

```suggestion
  time_t time;    ///< Timestamp from last engine status report [S]
```

ref: https://en.cppreference.com/w/c/chrono/time_t